### ISSUE TITLE
エラーの原因判別と解消がしたいです。

### ISSUE THREAD

**Issue by aiueo1234aiueo** [tablacus/TablacusExplorer#447]
環境はWindows10 home edditionにおいて、数ヶ月前のバージョンのタブラカスを使っていました。
アンチウィルスを切った後に最新版、安定版双方をダウンロードし起動しましたが全く同じエラーが出てしまいます。

エラーの前後で拡張プラグインの変更はしていません。
ローカルファイルからプラグインの判別をする方法が分からないので、導入しているプラグインが何かはわかりません。
よろしくおねがいします。

![スクリプト エラー 2021_06_29 23_20_37](https://user-images.githubusercontent.com/84449527/123816021-ed223900-d931-11eb-89cf-88211b1a3a06.png)
![about_blank 2021_06_29 23_21_24](https://user-images.githubusercontent.com/84449527/123816033-f1e6ed00-d931-11eb-8c44-a43d288c7b9d.png)

**Comment by tablacus** [comment]
「500 Internal Script Error」は起動時にスクリプトエラーがあると出ます。
原因としてはセキュリティソフト等に「js」ファイルが削除されてしまった場合に起こる場合がほとんどだと思います。
「LoadScripts」の本体が存在するのは「script」ファルダの「ui.js」なので存在するか確認してください。

※ここは多くの国の方が使われているので、タイトルだけは英語に変更していただけると幸いです。
本文の方は日本語でOKです。

**Comment by aiueo1234aiueo** [comment]
返信ありがとうございます。
ファイルの存在を確認しましたので、一応キャプチャをアップします。
![a](https://user-images.githubusercontent.com/84449527/123949042-11871f80-d9dd-11eb-8f00-6cc0ccba5065.png)
本文のみ日本語、承知しました。

**Comment by tablacus** [comment]
念のために当該フォルダをバックアップしてから以下の方法を試してみてください。

1. Tablacus Explorer 21.6.30をダウンロードしてzipを展開して上書きした場合はどうでしょう？
https://github.com/tablacus/TablacusExplorer/releases/tag/21.6.30

2.  Tablacus Explorer WebView2でエンジンをBlink化してみる。
https://tablacus.github.io/tewv2_ja.html

**Comment by tablacus** [comment]
LoadScriptsの段階でエラーが出ているので、アドオン読み込む前の最初の処理になります。
もしかすると、インターネットオプションを変更すると動くようになるかもしれません。

**Comment by aiueo1234aiueo** [comment]
半月ほど前のポイントへの「システムの復元」を行いました所、無事起動できました。

その前に、３つあげて頂いた方法を自分なりに試しましたが特に変化はありませんでした。
その際の手順は以下となります。
・最新verでの上書き
	セキュリティソフトを停止した後に最新verをダウンロード
	既存フォルダのバックアップを取った上で上記ファイルを上書き移動し、TE64.exeを起動
・Blink化
	WebView2 Runtimeのスタンドアロン版をダウンロード・インストールし、PCを再起動した後にTE64.exeを起動
・インターネットオプション
	「このゾーンのセキュリティレベル」中高→中
	「保護モード」無効

これからエラーが発生する直前への環境へ、エラーに留意しつつ戻していくつもりです。
またエラーに出くわした際にはこのスレッドへまた書き込ませて頂きます。
２日に渡って相談に乗って頂き、誠にありがとうございました。

**Comment by tablacus** [comment]
Blink化はいけると思ったのですが、システムの復元だったとは。
何にしても情報ありがとうございます。

**Comment by garry-ut99** [comment]
TE fresh setup doesn't run on Win 7 SP1, it throws 2 errors at startup and, then after 10 seconds, error No 3.

I solved/workarounded it by monkey-patching `script.js` by commenting out this section:

```js
if ("undefined" === typeof console) {
	console = {
		log: function () {
			const args = Array.apply(null, arguments);
			api.OutputDebugString(args.join("\n") + "\n");
		}
	}
}
```

Once commented out, TE runs fine, but I guess it should be patched more appropriately.

By the way, I also noticed that 2 errors on startup disappear after swhitching from IE engine to WV2 engine by copying dll files from: https://github.com/tablacus/TablacusExplorerWebView2/releases/tag/24.9.20 but the last "500" error still remains. (Until `script.js` is patched like mentioned above).

Screenshoots:


### 1
![Image](https://github.com/user-attachments/assets/1e9e8eb5-e23d-4358-a426-9f9f5cedd0ec)

### 2
![Image](https://github.com/user-attachments/assets/ba5aacd3-c5ed-4d09-9391-9f852d4930b5)

### 3
![Image](https://github.com/user-attachments/assets/71a0aa27-ffdf-4ce0-bbb9-06e678d6c278)

**Comment by ccfs** [comment]
@garry-ut99, the startup errors are probably due to old version of IE which has an old version of JS engine. I am also on Win7 SP1 but with IE11. I don't have the errors.

**Comment by garry-ut99** [comment]
Yeah, I was thinking about the same yesterday, I've updated from IE8 to IE11 and now TE starts fine.
